src/components/RichSelect/index.tsx

```diff
@@ -275,21 +363,13 @@ const SelectContainer = props => {
   )
 }
 
-SelectContainer.defaultProps = {
-  className: '',
-  getStyles: undefined,
-  innerProps: {},
-  isDisabled: false,
-  selectProps: {
-    mt: 0,
-  },
-}
 SelectContainer.propTypes = {
   children: PropTypes.node.isRequired,
   className: PropTypes.string,
-  getStyles: PropTypes.func,
-  innerProps: PropTypes.shape({}),
-  isDisabled: PropTypes.bool,
+  getStyles: PropTypes.func.isRequired,
+  innerProps: PropTypes.shape({}).isRequired,
+  isDisabled: PropTypes.bool.isRequired,
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
No way to remove this eslint rule ? Can't it be typed as `unknown` ?

Reply:
I wish it could...

Reply:
I tried:
- tweaking types
- type casting `as Validator<SelectProps>`
- type casting `as unknown`

nothing worked, typescript always complains about proptypes compatibility...